src/servers/history/run.ts

```diff
@@ -0,0 +1,56 @@
+import * as grpc from "@grpc/grpc-js"
+import healthCheck from "grpc-health-check"
+
+import { History } from "@app"
+import { ServiceStatus } from "@domain/index"
+import { baseLogger } from "@services/logger"
+
+import { protoDescriptorPrice } from "../grpc"
+
+const statusMap = {
+  "": 2,
```

Comment:
This is strange. Empty string?

Reply:
I have not check how works health service but this came from old code https://github.com/GaloyMoney/price/blob/ce60a47c816f20145c725a8c9aeb5e97187e744e/src/index.ts#L14